src/debug.c

```diff
@@ -1041,6 +1051,10 @@ void _serverAssertPrintClientInfo(const client *c) {
     serverLog(LL_WARNING, "client->conn = %s", connGetInfo(c->conn, conninfo, sizeof(conninfo)));
     serverLog(LL_WARNING, "client->argc = %d", c->argc);
     for (j = 0; j < c->argc; j++) {
+        if (!shouldRedactArg(c, j)) {
```

Comment:
I just started to look at this change (so maybe I am way off) but shouldn't we drop the negation here?

```suggestion
        if (shouldRedactArg(c, j)) {
```